client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
+		selectedSite: getSelectedSite( state ),
+		isPrivateSite: ( site ) => isPrivateSite( state, site.ID ),
```

Comment:
Would we create  a new function each time, based on the state and thus always rerender the component?